src/components/layout/hero/Main.js

```diff
@@ -62,6 +74,14 @@ const Main = () => {
                 >
                   <h2 style={{ color: "#deb887" }}>{item.title}</h2>
                   <h3>{item.subtitle}</h3>
+                  {item.type === "events" && (
+                    <>                                   
+                    <p>Date: {item.date ? item.date : ("15th April 2021")} Time: {item.time ? item.time : ("3:00PM")}</p>                    
```

Comment:
kiedy nie ma daty, to daj po prostu null, nie potrzebujemy czegoś bez daty, a data zawsze będzie z powodu walidacji. Destruktuj też obiekty... przykładem const {date, time} = item; kod będzie czytelniejszy.